app/presenters/question_presenter.rb

```diff
@@ -34,6 +34,7 @@ def hint
   end
 
   def caption
+    return nil if @renderer.hide_caption
```

Comment:
You don't need to specify the `nil`. That's the default behaviour or return:
```ruby
return if @renderer.hide_caption
```

Reply:
I think I'm going to leave the explicit nil there for readability.